api/test_app.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for get all entries"""
+import json
+import unittest
+
+
+from api import app
+
+"""Entries Request Response Tests"""
+
+
+class TestEndPoints(unittest.TestCase):
+    """Set Defaults"""
+
+    def setUp(self):
+        self.my_diary = app.app.test_client()
+        self.path = '/api/v1'
+
+    # The 4 main tests
+    def test_receive_entries(self):
+        """return all entries: GET /api/v1/entries"""
+        response = self.my_diary.get(self.path + '/entries')
+        self.assertEqual(response.status_code, 200)
+
+    def test_receive_specific_entry(self):
+        """return specific diary entry"""
+        _id = {"entry_id": 3}
+        response = self.my_diary.get(self.path + '/entries/{}'.format(_id['entry_id']))
+        self.assertEqual(response.status_code, 200)
+
+    def test_post_new_entry(self):
```

Comment:
The POST  method should be done to the url: `/api/v1/entries` not `/api/v1/entries/<int:entry_id>` . A user should not have to pass in an `id` to the POST. The id should be added automatically.

Reply:
I will fix it before progressing on. Thank You For the Feedback